src/asm/fstack.c

```diff
@@ -511,9 +513,9 @@ void fstk_RunFor(char const *symName, int32_t start, int32_t stop, int32_t step,
 	uint32_t count = 0;
 
 	if (step > 0 && start < stop)
-		count = (stop - start - 1) / step + 1;
+		count =  (uint32_t)((uint64_t)((int64_t)stop - start - 1) / step) + 1;
```

Comment:
I suspect this could be done with at least one fewer cast, but this is explicit about what it's doing, and it works as-is.